libs/eo/metering-points/shell/src/lib/eo-metering-points.store.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Injectable } from '@angular/core';
+import { ComponentStore } from '@ngrx/component-store';
+import { Observable } from 'rxjs';
+import {
+  EoMeteringPointsService,
+  MeteringPoint,
+} from './eo-metering-points.service';
+
+// Disabling this check, as no internal state is needed for the store.
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface EoMeteringPointsState {}
+
+@Injectable()
+export class EoMeteringPointsStore extends ComponentStore<EoMeteringPointsState> {
```

Comment:
Consider the setup of some basic error handling here - Component store has a baked in helper for that we can use, hint: tapResponse :-)

```suggestion
export class EoMeteringPointsStore extends ComponentStore<EoMeteringPointsState> {
meteringPoints$: Observable<MeteringPoint[]> = this.select(
    this.meteringPointsService.getMeteringPoints().pipe(
      tapResponse(
        (response) => response.meteringpoints,
        (error) => {
          // We only support the happy path for now
          throw error;
        }
      )
    ),
    (resp) => resp
  );
```

Verify this works :-D

Reply:
Added :) Testet that the error throws if the API returns 404